src/requests/team.ts

```diff
@@ -445,22 +445,31 @@ async function updateTeamSettings(
     gql`
       mutation UpdateTeamSettings(
         $team_id: Int
-        $teamSettings: team_settings_set_input!
+        $team_settings: team_settings_set_input!
+        $toggle_flows_offline: Boolean!
       ) {
         update_team_settings(
           where: { team_id: { _eq: $team_id } }
-          _set: $teamSettings
+          _set: $team_settings
         ) {
           returning {
             team_id
             id
           }
         }
+
+        # If a team is marked as "trial", toggle all flows offline
+        update_flows(
+          where: { team_id: { _eq: $team_id } }
+          _set: { status: offline }
+        ) @include(if: $toggle_flows_offline) {
```

Comment:
This `@include` directive is new to me and seems pretty handy!

Docs: https://hasura.io/docs/2.0/queries/postgres/variables-aliases-fragments-directives/#using-directives